### PR TITLE
Add initial HIPPA profile

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Add initial HIPPA profile

#### Rationale:

- https://bugzilla.redhat.com/show_bug.cgi?id=1513087

### CODE REVIEW

File: rhel7/profiles/hippa.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<Profile id="hippa">
+<title override="true">Health Insurance Portability and Accountability Act (HIPPA) Security Rule Profile</title>
```
Comment (shawndwells) [inline rhel7/profiles/hippa.xml]: Do not need "Security Rule Profile" in the title

File: rhel7/profiles/hippa.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<description override="true">
+The HIPAA Security Rule establishes national standards to protect individuals’
```
Comment (shawndwells) [inline rhel7/profiles/hippa.xml]: Would this be "HIPPA Security Rule" or "HIPPA profile" ?

Comment (shawndwells) [inline rhel7/profiles/hippa.xml]: wrt "national standards", lets define what national standards (US? EMEA? both?)

File: rhel7/profiles/hippa.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+electronic personal health information that is created, received, used, or
+maintained by a covered entity. The Security Rule requires appropriate
+administrative, physical and technical safeguards to ensure the
+confidentiality, integrity, and security of electronic protected health
+information. 
+
+This profile configures Red Hat Enterprise Linux 7 to the HIPPA Security
+Rule identified for securing of electronic protected health information.
```
Comment (shawndwells) [inline rhel7/profiles/hippa.xml]: single rule?

Comment (redhatrises) [reply rhel7/profiles/hippa.xml]: Yes. More to come. This was OTJ training on how to create a profile.

File: rhel7/profiles/hippa.xml (hunk 2 above)
Comment (redhatrises) [reply rhel7/profiles/hippa.xml]: It is actually referred to as the "HIPPA Security Rule"

File: rhel7/profiles/hippa.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<Profile id="hippa">
+<title override="true">Health Insurance Portability and Accountability Act (HIPPA)</title>
+<description override="true">
+The HIPAA Security Rule establishes U.S. national standards to protect individuals’
```
Comment (mpreisler) [inline rhel7/profiles/hippa.xml]: HIPAA vs HIPPA :)

**Pull request merged**